src/App.css

```diff
@@ -1,38 +1,50 @@
-.App {
+.app {
+  min-height: 100vh;
```

Comment:
I only know this from experience, but `100vh` can be kind of a pain, especially on mobile. What works really well is:

1. Setting height of 100% on the `html` and `body` and any nested non-primary container elements, in this case it would be on the `#root` 
2. Set a `min-height` of 100% on the main container, in this case `.app`

This will give you the same effect, as what you currently have, but it will be easier to maintain as more stuff gets added.

Reply:
Thank you so much for your review! We have fixed them. Please, If you have a little time, we'll appreciate your code review. 😄